### ISSUE TITLE
Problem with every time that open the app

### ISSUE THREAD

**Issue by YORDAN2223** [qnblackcat/uYouPlus#1560]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I have read the FAQ

### Device information

- iOS/iPadOS version: 16.7.10
- Device model: iPhone 8 Plus
- Sideload tool (AltStore, Sideloadly, TrollStore, etc.): Scarlett 

### IPA source

![IMG_1372](https://github.com/user-attachments/assets/5e016146-a9fe-48c7-bf9f-782ae55cdefe)

### Describe the issue

Every time that I open the app I see the screen like this 

### Steps to reproduce

1. 
2. 

### Are you using the latest version of uYouPlus? If not, why?

✅ Yes, I'm using the latest version of uYouPlus

### Does the issue happen with the official YouTube app from the App Store?

❌ No, I cannot reproduce this issue in the official YouTube app from the App Store

**Comment by apivovarov** [comment]
yes, it started to happen for me too several month ago. Also, when scrolling list of the videos I see this black screen too. Need to scroll down, wait till previews are loaded and scroll back up.

**Comment by Ahmed-Alashrafy** [comment]
Any update on this bug?

**Comment by lemvpt** [comment]
Same here, on iOS 18.0. Shorts turned off in preferences, but they still show up in default feed and cause these glitches: blank spaces that only populate when swiping back up.
